API/endpoints/CloudRecording.js

```diff
@@ -73,52 +73,90 @@ CloudRecording.listMeetingRecordings.scopes = [
  * @param {string} options.userId - the user ID or email address of the user
  * @param {number} [options.pageSize=300] - number of records
  *   returned from a single API call
- * @param {string} [options.nextPageToken=null] - token used to pageinate
+ * @param {string} [options.nextPageToken] - token used to pageinate
  *   through large result sets
  * @param {boolean} [options.searchTrash=false] - if truthy,
  *   include recordings from trash
  * @param {string} [options.trashType='meeting_recordings'] - Indicate type of
  *   cloud recording to retreive from the trash.
  *   options - {'meeting_recordings', 'recording_file'}
- * @param {string} [options.startDate=null] - query start date in
+ * @param {string|Date} [options.startDate] - query start date in
  *   'yyyy-mm-dd' format within last 6 months
- * @param {string} [options.endDate=null] - query end date in
+ * @param {string|Date} [options.endDate] - query end date in
  *   'yyyy-mm-dd' format within last 6 months
  * @return {Recording[]} List of Zoom Recordings {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/cloud-recording/recordingslist#responses}
  */
 CloudRecording.listUserRecordings = function (options) {
-  // Validate date strings if passed in
-  if (options.startDate && !utils.validateDate(options.startDate)) {
-    throw new ZACCLError({
-      message: 'Date needs to be in the format yyyy-mm-dd',
-      code: ERROR_CODES.INVALID_DATE_FORMAT,
-    });
+  // Destructure arguments
+  const {
+    userId,
+    searchTrash,
+  } = options;
+
+  let {
+    pageSize,
+    nextPageToken,
+    trashType,
+    startDate,
+    endDate,
+  } = options;
+
+  // If pageSize included, sanitize the value
+  if (pageSize) {
+    try {
+      // sanitizeInt throws standard Error if number invalid
+      pageSize = utils.sanitizeInt(pageSize);
```

Comment:
I made sanitizeInt throw a standard error so we can catch it and throw a more specific error since we might need to use sanitizeInt in more places in the future

Reply:
You can also catch ZACCLErrors if necessary